### PR TITLE
YLP-198 Review Platform project security scan

### PR DESCRIPTION
**Pull request by franck-fourel**
linked to https://github.com/mdblp/tools/pull/28

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,8 +59,8 @@ services:
   - docker
 
 script:
-  - make ci-generate ci-build ci-test ci-deploy ci-docker
-  - GO111MODULE="on" make ci-soups
+  # - make ci-generate ci-build ci-test ci-deploy
```
Comment (BigFab) [inline .travis.yml]: shall we remove this line completely now?

File: artifact.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  rm -f artifact_go.sh
+fi
+wget -q -O artifact_go.sh 'https://raw.githubusercontent.com/mdblp/tools/feature/add-platform-project/artifact/artifact_go.sh'
+chmod +x artifact_go.sh
+
+export ARTIFACT_GO_VERSION='1.11.4'
+
+# make sure build.sh can be executed 
+chmod 755 build.sh
+# Disable binary deployment in artifact_go.sh
```
Comment (BigFab) [inline artifact.sh]:
```suggestion
# Disable binary deployment in artifact_go.sh as it is done by the Make command
```

File: artifact.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+if [ -f artifact_go.sh ]; then
+  rm -f artifact_go.sh
+fi
+wget -q -O artifact_go.sh 'https://raw.githubusercontent.com/mdblp/tools/feature/add-platform-project/artifact/artifact_go.sh'
+chmod +x artifact_go.sh
+
+export ARTIFACT_GO_VERSION='1.11.4'
+
+# make sure build.sh can be executed
```
Comment (BigFab) [inline artifact.sh]:
```suggestion
```

File: artifact.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -ex
+
+if [ -f artifact_go.sh ]; then
+  rm -f artifact_go.sh
+fi
+wget -q -O artifact_go.sh 'https://raw.githubusercontent.com/mdblp/tools/feature/add-platform-project/artifact/artifact_go.sh'
```
Comment (BigFab) [inline artifact.sh]: will need to be reviewed once "Tools" PR is approved (https://github.com/mdblp/tools/pull/28)

**Review by BigFab — CHANGES_REQUESTED** [review]
Overall it's OK for me although this needs to be reviewed once the Tools PR is merged (https://github.com/mdblp/tools/pull/28/)

File: .travis.yml (hunk 1 above)
Comment (franck-fourel) [reply .travis.yml]: yep, it can be removed

**Review by BigFab — APPROVED** [review]
LGTM waiting for build before merging

**Pull request merged**